javascript/src/seedsource/containers/SeedZoneChooser.js

```diff
@@ -6,7 +6,7 @@ const mapStateToProps = ({ runConfiguration }) => {
     let { method, species, point, zones } = runConfiguration
     let { selected, matched, isFetchingZones } = zones
 
-    let pointIsValid = point !== null && point.x !== null && point.y !== null
+    let pointIsValid = point !== null && point.x !== null && point.y !== null && point.x !== "" && point.y !== ""
```

Comment:
Is empty string a valid state for `point.x` and `point.y`?

Reply:
Yes, through testing I could get empty strings but could never get null. Is null possible? (It looks like it's initialized with empty strings)